core/src/mindustry/game/SpawnGroup.java

```diff
@@ -2,6 +2,7 @@
 
 import arc.struct.*;
 import arc.util.*;
+import arc.Events;
```

Comment:
```suggestion
import arc.*;
```

Use wildcard imports.